superset/assets/src/SqlLab/reducers/sqlLab.js

```diff
@@ -318,6 +318,7 @@ export default function sqlLabReducer(state = {}, action) {
       const databases = {};
       action.databases.forEach((db) => {
         databases[db.id] = db;
+        databases[db.id]['allows_subquery'] = true;
```

Comment:
does every db allow subqueries? Shouldn't this be coming from the backend?

Reply:
maby not all,